views/tenants/index.erb

```diff
@@ -1 +1,11 @@
-Listing tenants for apartment #3
+<h2>Apartment at <%= @apartment.address %></h2>
+<h3>Tenants</h3>
+<ul>
+  <% tenants = @aparment.tenants %>
+  <%if tenants.size != 0%>
+    <% tenants.each do |tenant| %>
+      <li><%= tenant %></li>
```

Comment:
This will probably output something like `<Tenant::ActiveRecord::Base blahblahblah>`. You need to be more specific, like `tenant.name`.